Assets/HoloToolkit/Utilities/README.md

```diff
@@ -14,6 +14,16 @@ A drop in direction indicator that stays in the users view at all times.
 #### HeadsUpDirectionIndicatorPointer.prefab
 A quad based pointer to be used with the HeadsUpDirectionIndicator prefab to create an out of box direction indicator.
 
+###[Scenes](Scenes)
+---
+
+####HeadsetAdjustment.unity
```

Comment:
Also, just to make sure: this scene is intended to be used similar to a "Loading" scene, not as a starting point for an entire app. The real app will be a separate scene after this one.